src/resources/views/squads/edit.blade.php

```diff
@@ -50,6 +50,19 @@
                     </select>
                   </div>
                 </div>
+                <div class="form-group row">
+                  <label for="squad-classified" class="col-sm-1 col-form-label">Classified?</label>
+                  <div class="col-sm-1">
+                    @if($squad->is_classified)
+                      <input type="checkbox" name="classified" id="squad-classified" class="form-check-input ml-0" checked />
+                    @else
+                      <input type="checkbox" name="classified" id="squad-classified" class="form-check-input ml-0" />
+                    @endif
+                  </div>
+                  <div class="col-sm-10">
+                    <div>If a squad is classified, then only moderators will be able to see the member list</div>
```

Comment:
maybe add "moderators and administrators" here ?

Reply:
yeah.... added the admin viewing later... added, will commit shortly